recipes/lfpykit/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "LFPykit" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 72336a2332cd619b4e9d283995509356ff058b01234a72c024ec4d784877fc01
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.6
+    - pip
+    - numpy
```

Comment:
Since this is pure Python, you should only need `python` and `pip` under `host`, everything else should be under `run` (replacing `pin_compatible('numpy')`)

Reply:
Thanks, should be addressed now.